### PR TITLE
Clean up grammar in README

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 # Example usage
 
-Configuration for proxy and resolver can be done via a config file that can be specified with `-c config.toml`. The default configuration can be found at `tests/config.toml` and doesn't use a proxy, and uses https://odoh.cloudflare-dns.com aka 1.1.1.1 as the target resolver.  
+The proxy and resolver are configured using the file specified by the `-c` flag, e.g., `-c config.toml`. The default configuration can be found at `tests/config.toml`. It uses https://odoh.cloudflare-dns.com, i.e., 1.1.1.1, as the target resolver. It does not use a proxy by default.
```
Comment (sudheesh001) [inline README.md:17]: I'd suggest:

```
The current implementation out of the box doesn't use a default proxy and performs ODoH requests to the target resolver directly. Please configure a proxy before using the client to achieve the privacy properties of ODoH.
```

Comment (cjpatton) [reply README.md:17]: Works for me.

Comment (chris-wood) [reply README.md:17]: @cjpatton can you please make this change?

Comment (cjpatton) [reply README.md:17]: Done.

**Review by chris-wood — APPROVED** [review]

**Pull request merged**